protractor-circleci-conf.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var config = require('./protractor-shared-conf').config;
+
+
+// Using SauceLabs.
+config.capabilities = undefined;
+config.sauceUser = process.env.SAUCE_USERNAME;
+config.sauceKey = process.env.SAUCE_ACCESS_KEY;
+config.multiCapabilities = [
+  capabilitiesForSauceLabs({
+    browserName: 'chrome',
+    platform: 'OS X 10.14',
+    version: '81'
```

Comment:
Chrome Stable just hit `83`.

Reply:
According to this https://saucelabs.com/platform/supported-browsers-devices SauceLabs only has Chrome 81 available right now.